### ISSUE TITLE
terraform crash.log

### ISSUE THREAD

**Issue by Yucheol-Kim** [IBM-Cloud/terraform-provider-ibm#320]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.
Terraform version: 0.11.7

### Affected Resource(s)
Please list the resources as a list, for example:
- ibm_compute_instance

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```
https://gist.github.com/Yucheol-Kim/8394e844c4f3164365c4dad747b2391e

### Debug Output
Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.
* ibm_compute_vm_instance.vm-group[63]: ibm_compute_vm_instance.vm-group.63: connection is shut down
* ibm_compute_vm_instance.vm-group[72]: ibm_compute_vm_instance.vm-group.72: connection is shut down
* ibm_compute_vm_instance.vm-group[97]: ibm_compute_vm_instance.vm-group.97: unexpected EOF
* ibm_compute_vm_instance.vm-group[57]: ibm_compute_vm_instance.vm-group.57: connection is shut down
* ibm_compute_vm_instance.vm-group[74]: ibm_compute_vm_instance.vm-group.74: connection is shut down
* ibm_compute_vm_instance.vm-group[98]: ibm_compute_vm_instance.vm-group.98: connection is shut down
* ibm_compute_vm_instance.vm-group[91]: ibm_compute_vm_instance.vm-group.91: connection is shut down

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.
https://gist.github.com/Yucheol-Kim/bcd5da4e5e5ebc147eaee84ee6bd9991

### Expected Behavior
What should have happened?
I tried to order 300 servers divided into 3 times.

### Actual Behavior
What actually happened?
Some of my server orders have failed a lot.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`
terraform apply -auto-approve -parallelism=100

### Important Factoids
Are there anything atypical about your accounts that we should know? For example: Running in EC2 Classic? Custom version of OpenStack? Tight ACLs?

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-1234

**Comment by hkantare** [comment]
Fixed in v0.11.2 Please try with latest
https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v0.11.2

**Comment by hkantare** [comment]
Can you please share a detailed log when ever you see this issue..

**Comment by arielsalvo** [comment]
I've been running some tests and found out that the issue appears in different resources on different runs and that issuing a plan with a target parameter causes the run to finish correctly.
I was unable to get a detailed log of a failed run with a reasonable amount of resources since limiting this amount makes the issue go away.

**Comment by arielsalvo** [comment]
Appears to have been fixed at least since terraform v0.11.10/provider-ibm v0.12.0
Haven't seen the issue since upgrading to those versions a few weeks ago.
